_posts/2023-04-26-quarkus-3-0-final-released.adoc

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: 'Quarkus 3.0 released'
+date: 2023-04-26
+tags: release
+synopsis: 'It is our great pleasure to announce our new major release of Quarkus: Quarkus 3.0.'
+author: gsmet
+---
+
+It is our great pleasure to announce the release of Quarkus 3.0,
+our new major release,
+which comes packed with new features and enhancements.
+
+The development of Quarkus 3 started on March 18 last year with the crazy idea
+that we could rewrite the whole tree to migrate to the new `jakarta.*` packages
+while continuing the development of Quarkus 2.
+
+Quarkus 3 is now based on Jakarta EE 10 and, as you can expect,
+it is not the only change, this version comes packed with new features and enhancements
+that we implemented during this period.
+
+This announcement contains a lot of information about the new features and how to upgrade
+so we recommend to read it fully before upgrading to Quarkus 3.
+
+== What's new?
+
+=== Jakarta EE 10
+
+Quarkus 2 was based on Jakarta EE 8, Quarkus 3 is based on Jakarta EE 10.
+
+The most visible change is that the `javax.*` packages from Jakarta EE has been moved to `jakarta.*`.
+But Jakarta EE 10 also comes with new features for your favorite APIs.
+
+[NOTE]
+====
+Only the `javax.*` packages from Jakarta EE have changed.
+The ones from the JDK are still under `javax.*`.
+====
+
+=== Eclipse MicroProfile 6
+
+TODO
+
+=== Hibernate ORM 6
+
+This is one of the biggest changes in Quarkus 3: we upgraded to Hibernate ORM 6, more precisely 6.2.
+
+Hibernate ORM 6 is a major new version of Hibernate ORM and it comes with a lot of changes,
+with some being breaking changes.
+
+To learn more about the changes in Hibernate ORM 6, please have a look at the announcements for:
+
+- https://in.relation.to/2022/03/31/orm-60-final/[Hibernate ORM 6.0]
+- https://in.relation.to/2022/06/14/orm-61-final/[Hibernate ORM 6.1]
+- https://in.relation.to/2023/03/30/orm-62-final/[Hibernate ORM 6.2]
```

Comment:
Do we really want to point users to three 3 different links? I thought we had a Wiki page, no?

Reply:
agree - keep it simple first, details can be found for those who need to go deep.